fetch-schema.js

```diff
@@ -8,11 +8,5 @@ const thegraphConfig = require('./relay_thegraph.config')
 const THEGRAPH_API_URL = 'https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v3'
 exec(`get-graphql-schema ${THEGRAPH_API_URL} > ${thegraphConfig.schema}`)
 
-const API_URL = process.env.REACT_APP_GQL_API_URL
-const API_KEY = process.env.REACT_APP_GQL_API_KEY
-
-if (API_URL && API_KEY) {
-  exec(`get-graphql-schema ${API_URL} --h X-API-KEY=${API_KEY} > ${dataConfig.schema}`)
-} else {
-  console.log('REACT_APP_GQL_API_URL or REACT_APP_GQL_API_KEY is missing from env.local')
-}
+const DATA_API_URL = 'https://api.uniswap.org/v1/graphql'
```

Comment:
nit: should this use the process.env variable still?

Reply:
fair point, fixed